packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -139,7 +139,10 @@ export default function CodeBlock({
                   });
 
                   if (highlightLines.includes(i)) {
-                    lineProps.className += ' docusaurus-highlight-code-line';
```

Comment:
I'm not sure if we should keep this stable class, eg to make it easier adding custom styling.

Reply:
I think we should. In #7178 I'll allow any type of magic comments to be mapped to custom class names, so I think solely for consistency we should keep this class name.

Reply:
Then maybe move its definition to theme-common? https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-theme-common/src/utils/ThemeClassNames.ts

Reply:
Yes, I think that makes sense 👍

Reply:
Renamed `docusaurus-highlight-code-line` to `theme-code-block-highlighted-line` to be more consistent with the current approach.

Reply:
Actually, while the rename is fine, adding it to ThemeClassNames is unnecessary because in #7178 it will be injected from theme config...

We might change that API later though, so just keep it like this now.